.github/CODEOWNERS

```
@@ -289,9 +289,9 @@ pkgs/development/python-modules/buildcatrust/ @ajs124 @lukegb @mweinelt
 /nixos/tests/matrix-conduit.nix                            @piegamesde
 
 # Dotnet
-/pkgs/build-support/dotnet          @IvarWithoutBones
-/pkgs/development/compilers/dotnet  @IvarWithoutBones
-
+/pkgs/build-support/dotnet          @IvarWithoutBones @mdarocha @corngood @raphaelr @JamieMagee
+/pkgs/development/compilers/dotnet  @IvarWithoutBones @mdarocha @corngood @raphaelr @JamieMagee
+/pkgs/test/dotnet                   @IvarWithoutBones @mdarocha @corngood @raphaelr @JamieMagee
```

```suggestion
/pkgs/test/dotnet                   @IvarWithoutBones @mdarocha @corngood @raphaelr @JamieMagee
/doc/languages-frameworks/dotnet.section.md @IvarWithoutBones @mdarocha @corngood @raphaelr @JamieMagee
```

Please don't forget to also maintain related documentation. @NixOS/documentation-team is glad to assist with global questions.

---

Also, wouldn't it be easier to have a GitHub team and manage membership there? At least it would be a single source of truth.

---

> I assumed the teams.nix file creates github teams automatically?

That would very much surprise me, but I have no idea about how this works behind the scenes.

---

It doesn't, the teams.nix is extra metadata for various Nixpkgs scripts. You still need to create the team separately.

---

> I believe commit access is handled per-contributor.

It *could* be managed by giving a maintainer team write access and then having someone responsible to curate team membership. There is no rule in place at the moment, as far as I know.

---

> Maybe @RaitoBezarius or somebody else can shine more light on this, but I don't think any NixOS team have write access? I believe commit access is handled per-contributor.

Yes, I'm not aware of that, do you really need the constant pinging for everything in that tree? If not, consider waiting to have enough work to ask for write access and add only the active members of the .NET in that file.

---

Got it. :)